javascript/javascript2/week3/fastestGame/script.js

```diff
@@ -0,0 +1,40 @@
+console.log('Just test');
+function finishGame() {
+  let myBtn = document.getElementById('gameBtn');
+  myBtn.addEventListener('click', () => {
+    const time = document.getElementById('textInput').value;
+    setTimeout(() => {
+      const text = document.getElementById('overText');
+      text.innerHTML = 'Game over!!!';
+      if (keyboardS < keyboardL) {
+        alert('LLLLL winnnn !!!');
+      } else if (keyboardL < keyboardS) {
+        alert('SSSSSS win!!!');
+      } else if (keyboardS === keyboardL) {
+        alert('DRAW!!! try again');
+      } else {
+        return text;
+      }
+    }, time * 1000);
+  });
+}
+finishGame();
+
+let keyboardS = 0;
+let keyboardL = 0;
+//Keyboard event
+window.addEventListener('keypress', function (key) {
+  // console.log(key.keyCode);
+  if (key.keyCode === 115) {
```

Comment:
you can also use .key instead of .keycode, which gives you the string of the key pressed and you can compare it with the letters! using keycode also doesn't work in some browsers so it's better to avoid using it!